lib/liquid_voting_web/resolvers/delegations.ex

```diff
@@ -66,6 +66,9 @@ defmodule LiquidVotingWeb.Resolvers.Delegations do
 
       {:ok, delegation}
     else
+      {:error, message: message, details: details} ->
```

Comment:
I used the most basic error form for the error message `{:error, message: message, details: details}`, since this is the format we want all error messages to take. Unfortunately this means that, for now, I have added another error match in `resolvers/delegations.ex/create_delegation/3` to pass this error onward. These error pattern matches should be greatly reduced when #123 is addressed.

Reply:
true, this is getting out of hand 😅 but good for now